app/helpers/snowcrash_helper.rb

```diff
@@ -13,18 +13,18 @@ def css_class_for_sub_nodes(node)
   end
 
   def flash_messages
-    flash.collect do |name, msg|
-      flash_css = 'alert'
+    # In general controllers use :error, but :alert is used with redirect_to
+    #   http://guides.rubyonrails.org/action_controller_overview.html#the-flash
+    alert_types = {
+      'alert'   => ' alert-error',
+      'error'   => ' alert-error',
+      'info'    => ' alert-info',
+      'notice'  => ' alert-success',
+      'warning' => ''
+    }
 
-      # In general controllers use :error, but :alert is used with redirect_to
-      #   http://guides.rubyonrails.org/action_controller_overview.html#the-flash
-      flash_css << {
-        'alert'   => ' alert-error',
-        'error'   => ' alert-error',
-        'info'    => ' alert-info',
-        'notice'  => ' alert-success',
-        'warning' => ''
-      }[name]
+    flash.select { |key, _| alert_types.keys.include?(key) }.collect do |name, msg|
```

Comment:
The point is to allow keys in the flash that aren't about Bootstrap alerts. Elsewhere in this PR I'm using `:update_conflicts_since` as a flash key and in the existing `SnowcrashHelper` using anything except 'alert', 'error' etc as a flash key will raise an error.